internal/vendor/wiz.go

```diff
@@ -2117,6 +2130,13 @@ type CloudAccountConnection struct {
 	TotalCount int                 `json:"totalCount"`
 }
 
+// Users struct
+type Users struct {
```

Comment:
This is representative of UserConnection from the GraphQL schema.

The users Query from the GraphQL schema returns a UserConnection:

```
type Query {
  users(after: String, filterBy: UserFilters, first: Int): UserConnection!
}
```

This should be renamed to match the GraphQL type to aid in future development and avoid namespace conflicts.

```
type UserConnection {
  edges: [UserEdge!]
  exportUrl(format: ExportFormats = CSV, limit: Int = 5000): String
  nodes: [User!]
  pageInfo: PageInfo!
  totalCount: Int!
}
```

```suggestion
// UserConnection struct
type UserConnection struct {
```

Reply:
Good catch, will update